daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dfdlx.xsd

```diff
@@ -76,6 +77,23 @@
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:attribute name="inlineWhiteSpace" type="dfdlx:InlineWhiteSpaceEnum" default="disallow">
```

Comment:
I'm not sure I understand the name `inlineWhiteSpace`. This property doesn't really control whether or not inline white space is allowed, it really controls whether or not you can use these new character classes.

As I review this, I even wonder if a property to control this is needed? Doesn't the fact that a schema is using `%LSP*;` mean they want/need the capability enabled? And it doesn't help with compatibility with other DFDL implementations because they aren't going to know about the new dfdlx:inlineWhiteSpace extension since that's specific to Daffodil. Are we over complicating things by adding this property?